docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/policies.md

```diff
@@ -0,0 +1,125 @@
+---
+title: v4 code migration - Policies - Strapi Developer Docs
+description: Migrate policies from Strapi v3.6.8 to Strapi v4.0.x with step-by-step instructions
+canonicalUrl:  Used by Google to index page, should start with https://docs.strapi.io/ — delete this comment when done [paste final URL here]
+---
+
+<!-- TODO: update SEO -->
+
+# v4 code migration: Updating policies
+
+!!!include(developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/code-migration-intro.md)!!!
+
+::: strapi v3/v4 comparison
+
+In both Strapi v3 and v4, policies handle authorization. Policies can be global or scoped (i.e. applied to a specific API or plugin) and can be applied to REST controller's actions or to GraphQL resolvers.
+
+In Strapi v3, policies are Koa middlewares accepting or rejecting requests based on the REST context. As middlewares, v3 policies always receive a Koa context, either coming from the REST request or built from the GraphQL resolver arguments.
+
+In Strapi v4, [policies](/developer-docs/latest/development/backend-customization/policies.md#policies) are functions that should return `true` or `undefined` for the request to be accepted. v4 policies receive a custom context based on where the policy has been called from (e.g. a REST controller’s action or a GraphQL resolver).
```

Comment:
It should be noticed clearly that the key difference is you **cannot modify the custom context in a v4 policy** it's a read-only copy used for validation.

If you need context modification you need to use route middlewares.